analysed-packages/OpenSSL/version-1.1.1q/README.md

```diff
@@ -2,6 +2,10 @@
 
 https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1q.tar.gz
 
+## Package URL (purl)
+
+pkg:none/github.com/openssl/OpenSSL@1.1.1q
```

Comment:
`none` is not a known package type.
`pkg:github/openssl/openssl@1.1.1q` would be correct here. (note that the repo name is lowercase)